resources/views/date/gig/listAttendances.blade.php

```diff
@@ -88,19 +127,22 @@
 @section('js')
     <script type="text/javascript">
         $(document).ready(function () {
-            $(".btn-toggle").click(function () {
-                var voice = $(this).data('voice');
+            // open and close subvoices
+            $(".subvoice").click(function (){
+                var togglebutton = $(this).find(".btn-toggle");
+                var voice = togglebutton.data('voice');
 
-                //TODO: Make this more beautiful.
-                if ($(this).data("status") === "hidden") {
-                    $(".voice-" + voice).show();
-                    $(this).data("status", "display").find("i").removeClass("fa-caret-right").addClass("fa-caret-down");
-                    $(".super-voice-" + voice).data("status", "display").find("i").removeClass("fa-caret-right").addClass("fa-caret-down");
-                } else {
-                    $(".voice-" + voice).hide();
-                    $(this).data("status", "hidden").find("i").removeClass("fa-caret-down").addClass("fa-caret-right");
-                    $(".super-voice-" + voice).data("status", "hidden").find("i").removeClass("fa-caret-down").addClass("fa-caret-right");
+                $(this).nextAll(".voice-"+voice).toggle();
+                if("hidden" === togglebutton.data("status")){
+                    togglebutton.data("status","display").children("i").removeClass("fa-caret-right").addClass("fa-caret-down");
                 }
+                else{
+                    togglebutton.data("status", "hidden").children("i").removeClass("fa-caret-down").addClass("fa-caret-right");
+                }
+            });
+            // open and close comments for one user
+            $("td").has(".comment-toggle").click(function(){
+                $(this).siblings().addBack().find(".full-comment").toggle();
             });
```

Comment:
I don't think it's a good design choice that the comment disappears when one tries to highlight it. I think the button is big enough and the onclick-function for the comment itself should be removed.